pandas/tests/frame/test_apply.py

```diff
@@ -1012,3 +1012,18 @@ def test_non_callable_aggregates(self):
         expected = df.size
 
         assert result == expected
+
+    def test_frame_row_and_column_aggregates(self):
+        df = DataFrame({'B': [4, 5, 6, 7, 8],
+                        'C': [7, 8, 9, 10, 11]},
+                       index=[1, 2, 3, 4, 5])
+
```

Comment:
rather than just adding a new test. ideally would like to systematically test all routines.
you can create a fixture
```
@pytest.fixture(params=[0, 1])
def axis(request):
    return request.param
```